packages/relay/src/RelayServer.ts

```diff
@@ -138,6 +148,17 @@ export class RelayServer extends EventEmitter {
     }
   }
 
+  statsHandler (): StatsResponse {
+    const now = Date.now()
+    const statsResponse: StatsResponse = { ...this.readinessInfo, totalUptime: now - this.readinessInfo.runningSince }
+    if (this.isReady()) {
```

Comment:
took me a while to understand: you update here the partial state, since last state change... maybe we can put a comment.)

Reply:
Done